### ISSUE TITLE
Staging Review finding: Closing modal moves focus to unexpected place

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104649]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** vfs-mhv-secure-messaging
**Product:** mhv-secure-messaging
**Feature:** Signature Settings

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/profile/personal-information](https://staging.va.gov/profile/personal-information) when you press "Remove" to remove a signature but then cancel the action or otherwise close the confirmation modal without making changes, keyboard focus goes to the Edit button. Typically we expect closing a modal to take you back to whatever element was used to open the modal --- in this case, the Remove button --- so the user can proceed from where they left off on the page.

The actual impact here is pretty small, since focus is going to the element right before the Remove button. But I think it's still worth updating.

<details><summary>Link, screenshot or steps to recreate</summary>

![Messages signature section on profile, with an Edit primary button and a Remove secondary button.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/102012-Brian-1.png)

</details>

## Recommended action

When canceling the action or otherwise closing the confirmation modal without making changes, send focus back to the Remove button.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by lichellebain** [comment]
https://jira.devops.va.gov/browse/MHV-68301

**Comment by bryan-riley-va** [comment]
Closed 3/13/2025
